bonk/serializers.py

```diff
@@ -84,17 +87,20 @@ def create_link(self, instance):
                 'vrf': instance['vrf']
             }, request=self.context.get('request'))
 
+
 def validate_vrf(value):
     try:
         VRFSerializer.get(vrf=value)
     except:
         raise serializers.ValidationError("vrf=%r doesn't exist" % value)
     return value
 
+
 class IPBlockSerializer(HistorySerializerMixin):
     id = serializers.CharField(required=False)
     tags = serializers.DictField(required=False)
     vrf = serializers.IntegerField(required=True, validators=[validate_vrf])
+    name = serializers.CharField(required=True)
```

Comment:
I would probably put name below length, just for consistency with other things. vrf/network/length are used so much together.